src/Employer/Employer.Web/Orchestrators/Part2/AboutEmployerOrchestrator.cs

```diff
@@ -63,10 +71,32 @@ public async Task<OrchestratorResponse> PostAboutEmployerEditModelAsync(AboutEmp
             return await ValidateAndExecute(
                 vacancy,
                 v => _client.Validate(v, ValidationRules),
-                v => _client.UpdateDraftVacancyAsync(vacancy, user)
+                async v =>    
+                {
+                    vacancy.EmployerDescription = null; // We don't want to save the description until submission.
```

Comment:
I don't believe it does but I thought it would be more confusing adding the field as it may not match what is shown on screen if the description is updated in another vacancy.